### ISSUE TITLE
Logistic Request Table

### ISSUE THREAD

**Issue by knoxz** [Funwayguy/BetterQuesting#103]
Crafting inside a "Logistic Request Table" from Logistic Pipes does not update or count towards the crafting progress from a crafting quest using the latest Project Ozone 2.

Better Questing 1.0.173
Logistic Pipes 0.9.3.128

**Comment by Cazadorsniper** [comment]
can you update Better Questing to Better Questing version: 2.0.177

**Comment by knoxz** [comment]
server side and/or clientside?
Also are they cross compatible with different versions?
Like if server has updated one and client has the old one, or vise versa? :-D

**Comment by knoxz** [comment]
cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: betterquesting/network/handlers/PktHandler

Guess some other mod is not compatible?

**Comment by Funwayguy** [comment]
Updating from a BQ1.x.x build to a BQ2.x.x build is not a good idea if the pack was built around BQ1 mostly because every expansion would need to be updated accordingly.

As for your actual issue, most machine based or automated crafting will not trigger the crafting event as they do not have a player to reference.

TLDR: Machines aren't players, and don't mix versions of BQ

**Comment by knoxz** [comment]
Its not automated crafting or machine based. Its just a custom crafting table.

**Comment by Funwayguy** [comment]
Well then the table wasn't programmed to fire Forge crafting events, nothing I can do about it from my side sorry.

**Comment by knoxz** [comment]
Sure, just wanted to let you know.
https://github.com/RS485/LogisticsPipes/blob/dev/common/logisticspipes/blocks/crafting/LogisticsCraftingTableTileEntity.java

You might be faster in recognizing if there is anything you can do on your side.
